test/e2e/common/container_probe.go

```diff
@@ -213,8 +212,6 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call  does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
 	ginkgo.It("should be restarted with a docker exec liveness probe with timeout ", func() {
-		// TODO: enable this test once the default exec handler supports timeout.
-		e2eskipper.Skipf("The default exec handler, dockertools.NativeExecHandler, does not support timeouts due to a limitation in the Docker Remote API")
```

Comment:
Nice :-)

Reply:
I've confirmed this test problem can be reproduced if not applying this PR change with https://github.com/kubernetes/kubernetes/pull/96178 like
```
Kubernetes e2e suite: [k8s.io] Probing container should be restarted with a docker exec liveness probe with timeout 

test/e2e/common/container_probe.go:214 Nov  3 20:43:35.606: pod container-probe-5360/busybox-40384554-8b2e-41ed-8bd2-8c432418dc52 - expected number of restarts: 1, found restarts: 0 test/e2e/common/container_probe.go:224
```
This looks good for me to fix the bug and looking forward to seeing this in v1.20.